__tests__/test.test.js

```diff
@@ -63,3 +63,45 @@ describe('GET /api', () => {
     })
 })
 })
+describe('GET /api/articles/:article_id', () => {
+    test('Status code 200 returned', () => {
+        return request(app)
+        .get('/api/articles/2')
+        .expect(200)
+    })
+    test('Request responds with an article in the correct format', () => {
+        const articleFormat = {
+        author: expect.any(String),
+        title: expect.any(String),
+        article_id: expect.any(Number),
+        body: expect.any(String),
+        topic: expect.any(String),
+        created_at: expect.any(String),
+        votes: expect.any(Number),
+        article_img_url: expect.any(String)
+        }
+        const expected = expect.objectContaining(articleFormat)
+        return request(app)
+        .get('/api/articles/2')
+        .then(({ body }) => {
+            expect(body.article).toEqual(expected)
```

Comment:
Nice one Andy - we could be more specific here though on this test. For a specific response by ID as we have access to the data we could expect the hard coded data from the test data file for this entry.